lib/cartopy/mpl/gridliner.py

```diff
@@ -29,7 +29,7 @@
 from cartopy.crs import Projection, _RectangularProjection
 
 
-degree_locator = mticker.MaxNLocator(nbins=9, steps=[1, 2, 3, 6, 15, 18])
+degree_locator = mticker.MaxNLocator(nbins=9, steps=[1, 1.5, 1.8, 2, 3, 6, 10])
```

Comment:
The values are documented as and required to start at 1 and end at 10; they only worked with mpl1.5 this way by chance. It does mean a change in the result, but they are consistent between mpl1.5 and mpl2.0.